app/views/zines/show.html.haml

```diff
@@ -2,7 +2,7 @@
   %ul.zine_back
     %li
       = link_to I18n.t(:zine_back), zines_path
-%article.zine.column.span-24{:id => "zine_#{@zine.id}"}
+%article.zine.show.column.span-24{:id => "zine_#{@zine.id}"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace